runtime/doc/lua.txt

```diff
@@ -2014,6 +2014,35 @@ uri_to_fname({uri})                                       *vim.uri_to_fname()*
 ==============================================================================
 Lua module: ui                                                        *lua-ui*
 
+confirm({msg}, {opts}, {on_choice})                         *vim.ui.confirm()*
+    Prompts the user to pick an option from a list of choices
+
+    Example: >lua
+
+     vim.ui.confirm("Favorite fruit?", {
+         choices = {'Apple', 'Banana', 'Cranberry', 'El&derberry' },
+         default = 2,
+     }, function(idx, choice)
+         print(idx, choice)
+     end)
+<
+
+    Parameters: ~
+      • {msg}        (string) prompt displayed to the user
+      • {opts}       (table)
+                     • choices (string[]|nil) List of choices. The shortcut
+                       key for each choice is designated with a "&" character.
+                       If "&" is not present in the choice, the first letter
+                       of the choice is used.
+                     • default (number|nil) Default choice selected when user
+                       presses <Enter>
+      • {on_choice}  fun(idx:number|nil,choice:string|nil) Called once the
+                     user chooses or cancel the input. `idx` is the 1-based
+                     index of the 'choice' in `opts.choices` table, (it might
```

Comment:
> Any idea why these tests don't work?

Because of this I think:

```suggestion
                     index of the `choice` in `opts.choices` table, (it might
```